src/templates/language_packs.html

```diff
@@ -0,0 +1,46 @@
+{% from "_macros/market_button.html" import market_button %}
+{% include '_macros/more_button.html' %}
+
+<section class="main">
```

Comment:
I filed https://bugzilla.mozilla.org/show_bug.cgi?id=1132148 which will make a lot of our templates more concise